src/VirtualClient/VirtualClient.Actions/StressAppTest/StressAppTestMetricsParser.cs

```diff
@@ -0,0 +1,82 @@
+﻿namespace VirtualClient.Actions
```

Comment:
Please add the MSFT MIT header for all the cs files you added.

Reply:
added the header